pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -332,6 +332,8 @@ Currently there are three different payload formats you can choose from for the
 
 Json containing a Snowplow Schema for every event included in the Publishing integration. These schemas can for example be used to integrate Avo with Snowplow Iglu.
 
+You can provide your own [vendor](https://docs.snowplowanalytics.com/docs/understanding-tracking-design/understanding-schemas-and-validation/#:~:text=%E2%80%9Cvendor%E2%80%9D%20%E2%80%93%20This%20usually%20refers,events%20and%20contexts%20(e.g.%20com.) value via the "Vendor" input in the publishing integration config.
```

Comment:
Hmm, this doesn't seem to be formatting correctly. it it missing a closing parenthesis?
<img width="854" alt="Screenshot 2022-03-04 at 09 28 28" src="https://user-images.githubusercontent.com/4923531/156736938-b5efa9b3-593f-457c-a1d9-752a39748d85.png">

Reply:
oops, great catch! fixed in 898bd48